### DISCUSSION TITLE
Warning: urban mask

### DISCUSSION THREAD

**Discussion by j3r3m1** [orbisgis/geoclimate#879] [General]
It seems that the urban area mask layer fails for areas having no urban tang for the area (it is more a problem in OSM than in GeoClimate). However it means we should carefully check the urban land mask before using it and if necessary add the missing informations in OSM.

Two examples in Dijon:
![image](https://github.com/orbisgis/geoclimate/assets/13120405/d70f175b-4e4a-4570-ae8a-bc470a7cadbe)

It seems there are "rural areas" in OSM but actually it is not the case:
Case 1 is "Palais des Congrès and co" area:
https://www.openstreetmap.org/query?lat=47.33163&lon=5.05382#map=18/47.32993/5.05397

Case 2 is south of the train station, nearby a park:
https://www.openstreetmap.org/query?lat=47.32066&lon=5.02378#map=17/47.32020/5.02606

In my opinion, this discussion does not mean we need to adjust GeoClimate but at least checking the urban_mark_layers we obtain using GeoClimate and modify OSM data if needed before to use this layer (or manually modify the GeoClimate output but this would be ashame not contributing to OSM.

@MGousseff I know you are using this data

**Comment by ebocher** [comment]
The urban_areas layer is used to tag buildings when there is no information on their type and to split the TSU. Because they are not continuous and well distributed, the urban_areas  cannot be used for analysis.
 It looks to me  @MGousseff  doesn't use this layer

A WARNING on the wiki page is a good idea.

**Reply by MGousseff** [reply]
@j3r3m1 In fact, what we are going to use is an urban area reconstructed from the LCZ post processing. To my knowledge, the algorithme of producing this urban area is still under construction. And I will definitely use it as soon as it is produced.

**Reply by j3r3m1** [reply]
My bad I totally forgot the fact that the urban area that you are using are based on LCZ output, which is a good news. I think we should indeed add an information about this urban mask in the documentation. @franetibe could you add such information ?

**Reply by franetibe** [reply]
Sure, I will add a warning, let me know what you think about it.

**Reply by j3r3m1** [reply]
It has been added here: https://github.com/orbisgis/geoclimate/wiki/Urban-areas-layer
It is OK for me, thanks @franetibe
